src/page/admin/auth/oidc-auth-container.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import OidcAuth from './oidc-auth';
+import { getOidcConfig, updateOidcConfig } from '../../../store/e-admin-auth/actions';
+
+const mapStateToProps = state => ({
+    config: state.authAdmin.get('oidc'),
+    unleashUrl: state.uiConfig.toJS().unleashUrl,
+});
+
+const Container = connect(mapStateToProps, { getOidcConfig, updateOidcConfig })(OidcAuth);
```

Comment:
This works. But giving a more explicit name is better for auto importers. When we have multiple components named Container my auto importer struggles to figure out which container to import. 

```suggestion
const OidcContainer = connect(mapStateToProps, { getOidcConfig, updateOidcConfig })(OidcAuth);
```